lib/px/utils/pxtimer.lua

```diff
@@ -13,26 +13,9 @@ function M.application(px_configutraion_table)
 	local px_logger = require ("px.utils.pxlogger").load(px_config)
 	local px_constants = require("px.utils.pxconstants")
 	local buffer = require "px.utils.pxbuffer"
-	local px_commom_utils = require('px.utils.pxcommonutils')
 
 	local ngx_timer_at = ngx.timer.at
 
-	function send_initial_enforcer_telemetry()
```

Comment:
why did you remove this?

Reply:
The feature states that we should stop sending enforcer telemetry on init, because in some states when there are many instances, it sends too many.